pkg/component/tools/compogen/pkg/gen/schema.go

```diff
@@ -15,6 +15,7 @@ type property struct {
 	Items struct {
 		Type       string              `json:"type"`
 		Properties map[string]property `json:"properties" validate:"omitempty,dive"`
+		OneOf []objectSchema `json:"oneOf" validate:"dive"`
```

Comment:
nit: it looks like `gofmt` hasn't been executed (typically it does when you save a file) on this struct 🤔